ntbs-service/Services/AlertService.cs

```diff
@@ -39,11 +39,14 @@ public async Task DismissAlertAsync(int alertId, string userId)
         {
             var alert = await _alertRepository.GetAlertByIdAsync(alertId);
 
-            alert.ClosingUserId = userId;
-            alert.ClosureDate = DateTime.Now;
-            alert.AlertStatus = AlertStatus.Closed;
+            if (alert != null)
+            {
+                alert.ClosingUserId = userId;

```

Comment:
`GetAlertByIdAsync` return open alerts only and therefore causing Null pointer exception when alert is closed